README.md

```diff
@@ -1 +1,20 @@
-# viziphant
+# Viziphant
+
+A Python module for easy visualization of Neo objects and Elephant results.
+
+Viziphant provides a high-level API to easily generate plots and interactive visualizations of neuroscientific data and analysis results. This API uses and extends the same structure as in Elephant to ensure intuitive usage for scientists that are used to Elephant. 
+
+<br><br>
+__Installation__
+
+For a development install follow these instructions:
+
+```
+git clone https://github.com/INM-6/viziphant.git
```

Comment:
Would `pip install git+https://github.com/INM-6/viziphant.git@master` also work as a shorter way of installing?

Reply:
I didn't know pip is able to do this. But I just tried and it seems to be working fine, so yes, you are right.
This will, however, not create a development installation that allows changes without reinstalling. Do you think it should be added already anyway?

Reply:
I think you may be right, or do you know if `pip install --upgrade` would work in this scenario? If not, maybe it's better to stick with your manual install instructions.

Reply:
`pip install --upgrade git+https://github.com/INM-6/viziphant.git@master` should work according to StackOverflow (https://stackoverflow.com/questions/17710947/pip-pulling-updates-from-remote-git-repository). But else pip seems use PyPI only without remembering where a package was installed from.

Reply:
I can confirm this works, but I agree we could leave it as is since it's labeled a development install.